styles/build.less

```diff
@@ -4,7 +4,7 @@
 .build {
 
   min-width: 250px;
-  font-family: Menlo;
+  font-family: monospace;
```

Comment:
I don't think this is required at all, since an actual font-family will be fetched from config, so generally user won't see a font defined here. I think it's fine to set some fancy fonts here just in case something really bad happens :)